JUSTFILE

```diff
@@ -25,7 +25,7 @@ load-cloudbeat-image:
 
 build-cloudbeat:
   GOOS=linux go mod vendor
-  GOOS=linux go build -v && docker build -t cloudbeat .
+  GOOS=linux go build -v && docker buildx build --platform=linux/amd64 -t cloudbeat .
```

Comment:
what about other platforms?

Reply:
This one will be reverted. I've added it when I experimented with EKS cluster.